dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -1609,16 +1611,19 @@ public ContentletHashMap() {
 			super();
 		}
 
-		public Object put(final String key, final Object value) {
+		public Object put(final String key, final Object newValue) {
 
-			Contentlet.this.markAsDirty();
+		    final Object oldValue = this.get(key);
+		    if(!java.util.Objects.equals(oldValue, newValue)) {
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.util.Objects.equals' due to existing import 'java.util.Objects'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6325390)